src/equations/numerical_fluxes.jl

```diff
@@ -305,12 +307,27 @@ end
 
 @inline function (numflux::FluxHydrostaticReconstruction)(u_ll, u_rr,
                                                           orientation_or_normal_direction,
-                                                          equations::AbstractEquations)
+                                                          equations::AbstractEquations{1})
   @unpack numerical_flux, hydrostatic_reconstruction = numflux
 
   # Create the reconstructed left/right solution states in conservative form
   u_ll_star, u_rr_star = hydrostatic_reconstruction(u_ll, u_rr, equations)
 
+  threshold = equations.threshold_wet
```

Comment:
I would add a comment here as to what the `threshold_wet` means and what its default value is.